float64[] query
---
float64 outcome